README.md

```diff
@@ -42,7 +42,10 @@ TODO:
 
 ## Development
 
-Simply spawn the server by running `$ npm run start:dev`.
+- set env variable 'BROWSER' in the json, dotenv file or before
```

Comment:
Instead of changing the readme, which still needs to be constructed, could you create a section in .env.sample similar to the other env variables?